### DISCUSSION TITLE
[request] optional option to make replys readable in chat

### DISCUSSION THREAD

**Discussion by lastmetroid** [night/betterttv#6930] [Ideas]
Please add one or two optional options you can activate to make replys readable in chat, because theyre not. the cut off with "..." in a reply in chat is a really bad idea for other people who want to see what was replied to. the mouse over is not helpful because of chat moving. suggestion:

1. give optional option so the cut off is removed and the reply is showing the entire message in chat which was replied to
2. give another optional option, clicking on a message shows the original message in a highlight or box which wont go away until you click close/x on it

**Comment by night** [comment]
Option 2 is already implemented on native Twitch. You can click on the Reply button to view the entire thread of messages.
